packages/plugins/execute-subscription-event/src/subscribe.ts

```diff
@@ -0,0 +1,49 @@
+import { createSourceEventStream } from 'graphql';
+import isAsyncIterable from 'graphql/jsutils/isAsyncIterable';
+import mapAsyncIterator from 'graphql/subscription/mapAsyncIterator';
+import { makeSubscribe } from '@envelop/core';
+import { ExecuteFunction, SubscribeFunction } from '@envelop/types';
+
+/**
+ * This is a almost identical port from graphql-js subscribe.
+ * The only difference is that a custom `execute` function can be injected for customizing the behavior.
+ */
+export const subscribe = (execute: ExecuteFunction): SubscribeFunction =>
```

Comment:
Some plugins might use `setExecuteFn` during `onExecute` with the assumption that they override this function.
Does it mean that these plugins need somehow to override this function here ?